Mimble Wimble.md

```diff
@@ -0,0 +1,146 @@
+
+## Peggingn-in Transactions
+
+To engage with this privacy feature, users perform a "pegging-in" transaction to move their Litecoin into an Extension Block, using a specific MimbleWimble Extension Block (MWEB) address. These transactions are facilitated by a special transaction known as the HogEx (Hogwarts Express), which processes both the pegging-in of funds to the Extension Block and the pegging-out back to the main, transparent blockchain. The HogEx also manages the Extension Block balance, adjusting it for incoming and outgoing transactions.
+
+While the majority of Litecoin transactions continue to operate on the transparent main blockchain, the introduction of MimbleWimble via Extension Blocks provides a significant step forward in offering privacy options to users.
+
+Implementing MimbleWimble on the Decred blockchain is theoretically possible. Decred governance model and adaptability could facilitate the consideration and potential implementation of MimbleWimble if there was enough community support. But we also need to ask if a MimbleWimble implementation would be necessary.
+
+Decred already has a great tool for keeping transactions scalable, private, low-cost, cheap, and near-instant, it's the Decred Lightning Network (LN). LN is a layer 2 protocol that runs on top of the base layer, a technological solution designed to solve transaction problems by introducing off-chain transactions.
+
+The technical integration of MimbleWimble would require a lot of development effort and a thorough review process to ensure that it aligns with Decred’s security, privacy, and scalability goals. One of the main considerations would be how to integrate MimbleWimble's privacy features with Decred's existing features, such as its governance system, and the Stakeshuffle protocol.
+
+There are also some Mimblewimble drawbacks that need to be taken into consideration: 
+
+Slower Transactions: Networks that use Confidential Transactions, like MimbleWimble, can be slower than others such as Ripple (XRP) and proof-of-stake networks like Solana and Cardano. This is because the data for Confidential Transactions is larger, which can slow things down.
+
+Quantum Computer Risk: MimbleWimble relies on digital signatures to work. This makes it potentially more at risk from future attacks by quantum computers, which might be able to break these digital signatures. Decred has overcome this problem with post-quantum secure mixing on stakeshuffle.
+
+## Moving Forward
+
+It's always interesting to learn about privacy protocols, especially one that is much commented like Mimblewimble. Do you think it could be a good addition to the Decred blockchain? Let your voice be heard in the comment section below!
+
```

Comment:
Just a few more small edits from me. It might also be worth seeing if someone with more knowledge of MW would give this a read through before we go to publish on CT.